clio-server/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/AutoElasticsearchQueryMapper.scala

```diff
@@ -26,9 +25,11 @@ import java.time.OffsetDateTime
   * @tparam Document         The Elasticsearch documents being queried, with a context bound also specifying that an
   *                          `implicit ctagDocument: ClassTag[Document]` exists.
   */
-class AutoElasticsearchQueryMapper[ModelQueryInput: ClassTag: FieldMapper,
-                                   ModelQueryOutput: ClassTag,
-                                   Document <: ClioDocument: ClassTag] private
+class AutoElasticsearchQueryMapper[
+  ModelQueryInput: ClassTag: FieldMapper,
+  ModelQueryOutput: ClassTag,
+  Document <: ClioDocument: ClassTag
+] private[dataaccess]
     extends ElasticsearchQueryMapper[
```

Comment:
Any chance scalafmt will let you push this `extends` up to the line above?

Reply:
No